joomla/joomla-cms
# Add a parameter to the configuration to allow the mailto link to be switched on or off

Pull Request replaces #9490

#### Summary of Changes
This PR allows to set the mailto link for each contact individually.

#### Test Instructions
1. **Backend:** Create a contact. Give it a title and make sure to fill out the email address field. The contact doesn't have to be assigned to a user or a category. For easy access, create a menu item to that contact in a menu of your choice.
2. **Frontend:** Verify that the email address including the mailto link is visible in the front-end. If not, please check the settings for com_contact.
3. **Backend:** Apply the patch, go to either the component, the contact or menu settings and set "Add Mailto: Link" to "No".
4. While the email address still is obfuscated, there should be no mailto: link.

### Documentation Changes Required
**Note: language files were changed and help screens might need to be updated.**
